src/aws/nist-800-53-rev4/rules/aws-nist-800-53-rev4-12.1.ts

```diff
@@ -0,0 +1,51 @@
+export default {
+  id: 'aws-nist-800-53-rev4-12.1',  
+  title: 'CloudFront distributions should have geo-restrictions specified',
+  
+  description: `CloudFront distributions should enable geo-restriction when an organization needs to 
+  prevent users in specific geographic locations from accessing content. For example, 
+  if an organization has rights to distribute content in only one country, geo restriction should be 
+  enabled to allow access only from users in the whitelisted country. Or if the organization cannot 
+  distribute content in a particular country, geo restriction should deny access from users in the 
+  blacklisted country.`,
+
+  audit: '',
+  
+  rationale: '',
+  
+  remediation: `**AWS Console**  
+
+  - Navigate to CloudFront.
+  - Select the distribution that you want to update.
+  - In the Distribution Settings pane, select the Restrictions tab > Edit.
+  - Enter the applicable values. For more information, refer to Restrictions.
+  - Choose Yes, Edit.
+  
+  **AWS CLI**
+  - Submit a GetDistributionConfig request to get the current configuration and an Etag header for the distribution.
+    - > get-distribution-config --id <value> 
+  - Update the returned XML to include the CloudFront should have geo-restrictions specified.
+  - Submit an UpdateDistribution request to update the configuration for your distribution. Refer to here for more information.`,
+  
+  references: [
+      'https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesRestrictions',
+      'https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/georestrictions.html',
+      'https://docs.aws.amazon.com/cli/latest/reference/cloudfront/update-distribution.html',
+  ],
+  gql: `{
+    queryawsCloudfront {
+      id
+      arn
+      accountId
+      __typename
+      geoRestrictions
+    }    
+  }`,
+  resource: 'queryawsCloudfront[*]',
+  severity: 'medium',
+  conditions: {
+    path: '@.geoRestrictions',
+    notIn: ['Blacklist']
```

Comment:
I am not sure if this is defined correctly. Can you share your thinking here? It looks like the rule is stating that `geoRestrictions` can not be set to "none" like seen below

<img width="433" alt="image" src="https://user-images.githubusercontent.com/7192733/166290064-dc9ead45-26a1-42d2-b815-074315bcc1b0.png">

Reply:
@tyler-dunkel Hello Tyler. When I was testing on the playground I saw the geoRestriction tag returning none, as in no country has been whitelisted or blacklisted. So in the off chance there is, any blacklisted country will fail the rule. But from your comment I can see my thought process is wrong. Its the none itself that should not be possible. Adding the 'none' label into the notIn should fix it. 
Thanks for review. I'll make sure to fix it.